django/db/backends/postgresql/schema.py

```diff
@@ -151,9 +158,35 @@ def _alter_field(self, model, old_field, new_field, old_type, new_type,
         # Removed an index? Drop any PostgreSQL-specific indexes.
         if old_field.unique and not (new_field.db_index or new_field.unique):
             index_to_remove = self._create_index_name(model._meta.db_table, [old_field.column], suffix='_like')
-            self.execute(self._delete_index_sql(model, index_to_remove))
+            self.execute(self._delete_index_sql(model, index_to_remove, concurrently=concurrently))
 
     def _index_columns(self, table, columns, col_suffixes, opclasses):
         if opclasses:
             return IndexColumns(table, columns, self.quote_name, col_suffixes=col_suffixes, opclasses=opclasses)
         return super()._index_columns(table, columns, col_suffixes, opclasses)
+
+    def add_index(self, model, index, concurrently=False):
+        """Add an index on a model."""
+        self.execute(index.create_sql(model, self, concurrently=concurrently), params=None)
+
+    def remove_index(self, model, index, concurrently=False):
+        """Remove an index from a model."""
+        self.execute(index.remove_sql(model, self, concurrently=concurrently))
+
+    def _delete_index_sql(self, model, name, concurrently=False):
+        return Statement(
+            self.sql_delete_index if not concurrently else self.sql_delete_index_concurrently,
+            table=Table(model._meta.db_table, self.quote_name),
+            name=self.quote_name(name),
+        )
+
+    def _create_index_sql(self, model, fields, *, name=None, suffix='', using='', db_tablespace=None, col_suffixes=(),
+                          sql=None, opclasses=(), condition=None, concurrently=False):
+        sql = self.sql_create_index if not concurrently else self.sql_create_index_concurrently
+        return super()._create_index_sql(
+            model, fields, name=name, suffix=suffix, using=using, db_tablespace=db_tablespace,
+            col_suffixes=col_suffixes, sql=sql, opclasses=opclasses, condition=condition,
+        )
+
+    def alter_index_together(self, model, old_index_together, new_index_together, concurrently=False):
+        super().alter_index_together(model, old_index_together, new_index_together, concurrently=concurrently)
```

Comment:
We can remove `alter_index_together()` because you are not able to alter index concurrently (sorry for confusing in my previous [comment](https://github.com/django/django/pull/11594#discussion_r307687220)).